airbyte-integrations/connectors/source-jina-ai/source_jina_ai/manifest.yaml

```diff
@@ -0,0 +1,158 @@
+        type: HttpRequester
+        url_base: "https://r.jina.ai/{{ config['read_prompt'] }}"
+        http_method: "GET"
+        path: "/"
+        authenticator:
+          type: BearerAuthenticator
+          api_token: "{{ config['api_key'] }}"
+        request_headers:
+          Accept: application/json
+          X-With-Links-Summary: "{{ config['gather_links'] }}"
+          X-With-Images-Summary: "{{ config['gather_images'] }}"
+    name: "reader"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/schemas/reader"
+
+  search_stream:
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        $ref: "#/definitions/selector"
+      paginator:
+        type: NoPagination
+      requester:
+        type: HttpRequester
+        url_base: "https://s.jina.ai/{{ config['search_prompt'] }}"
+        http_method: "GET"
+        path: "/"
+        authenticator:
+          type: BearerAuthenticator
+          api_token: "{{ config['api_key'] }}"
+        request_headers:
+          Accept: application/json
+          X-With-Links-Summary: "{{ config['gather_links']}}"
+          X-With-Images-Summary: "{{ config['gather_images']}}"
+    name: "search"
+    $parameters:
+      path: "/example"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/schemas/search"
+
+streams:
+  - "#/definitions/reader_stream"
+  - "#/definitions/search_stream"
+
+check:
+  type: CheckStream
+  stream_names:
+    - "reader"
+    - "search"
+
+schemas:
+  search:
+    $schema: http://json-schema.org/draft-07/schema#
+    title: Jina AI URL Reader Schema
+    type: object
+    additionalProperties: true
+    properties:
+      title:
+        type: string
+      url:
+        type: string
+      content:
+        type: string
+      description:
+        type: string
+      links:
+        type: object
+        additionalProperties: true
+        properties:
+          More information...:
+            type: string
+
+  reader:
+    $schema: http://json-schema.org/draft-07/schema#
+    title: Jina AI prompt Reader Schema
+    type: object
+    additionalProperties: true
+    properties:
+      title:
+        type: string
+      url:
+        type: string
+      content:
+        type: string
+      description:
+        type: string
+      links:
+        type: object
+        additionalProperties: true
+        properties:
+          More information...:
+            type: string
+
+spec:
+  type: Spec
+  documentation_url: https://docs.airbyte.com/integrations/sources/jina-ai
+  connection_specification:
+    title: Jina Ai Spec
+    type: object
+    required:
+      - api_key
+    additionalProperties: true
+    properties:
+      api_key:
+        type: string
+        title: API Key
+        description: >-
+          Your Jina AI API Key. See <a
+          href="https://jina.ai/reader/#apiform">here</a>.
+        airbyte_secret: true
```

Comment:
Is it possible to make this optional? I think we get a lower rate limit for the requests, but it can streamline POCs and smaller use cases if so.

Reply:
:white_check_mark: